### DISCUSSION TITLE
Will renovatebot try to update a dependency to the latest version if it has an open CVE?

### DISCUSSION THREAD

**Discussion by diogoteles08** [renovatebot/renovate#26238] [Request Help] [answered]
### How are you running Renovate?

None

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

Let's say I have renovatebot installed with a monthly schedule, so I receive my renovate PR on the 1st of each month.

One of my dependencies releases a new version on the 28th and is soon found to have a malicious security vulnerability (the maintainer's account was hijacked, for example) on the 29th. The project's real maintainer is unable to access their repo or account in the package index to revert the malicious version, but manages to publish a CVE/GHSA/OSV for the malicious version on the 30th.

When renovatebot runs on my project on the 1st, will I receive a PR updating me to the malicious version or does renovatebot first confirm that the version is "safe"?

**Comment by rarkins** [comment, marked as answer]
No, but this is a highly unlikely scenario because malicious packages are usually removed from registries long before a CVE is issued. Anyone can report malicious packages, and registries attempt to remove them with a high urgency. Meanwhile CVE issuing and distribution is a complicated process.
